test/unit/SourceLocator/SourceStubber/PhpStormStubsSourceStubberTest.php

```diff
@@ -265,6 +271,17 @@ public static function internalFunctionsProvider(): array
             array_filter(
                 $functionNames,
                 static function (string $functionName): bool {
+                    if (
+                        in_array($functionName, [
+                        // Missing in JetBrains/phpstorm-stubs
```

Comment:
Are the stubs to be updated in `composer.lock`?

Reply:
We use latest version currently. So we have to wait for new release.